### PR TITLE
Removes turnstiles

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

Removes turnstiles from all maps and removes the file.

## How This Contributes To The Skyrat Roleplay Experience

These are no longer maintained and are not really needed as we already have turnstiles.

Also
image

## Changelog

:cl:
del: Turnstiles have been removed.
/:cl:

### CODE REVIEW

File: _maps/RandomRuins/SpaceRuins/skyrat/interdynefob.dmm [hunk 1]
```diff
@@ -1596,9 +1596,8 @@
 /area/template_noop)
 "fH" = (
 /obj/machinery/door/firedoor,
-/obj/machinery/turnstile{
-	dir = 1;
-	req_one_access_txt = "151"
+/obj/machinery/door/airlock/security{
+	req_access = list("brig")
```
Comment (Jolly-66) [inline _maps/RandomRuins/SpaceRuins/skyrat/interdynefob.dmm]: We use access helpers for airlocks, not `req_access`.

**Review by Jolly-66 — COMMENTED** [review]
I'm going to be annoying.

If it isn't a bother, please, either here or send it to me, an update paths since I'm currently working on resetting the maps. Not that it should create much of an issue, but saving me a headache would be most appreciated.

**Comment by Iamgoofball** [comment]
Did you replace these with the version from Tramstation?

**Comment by Jolly-66** [comment]
> Did you replace these with the version from Tramstation?

I think those got lost in a reset. CI isn't crying.

**Pull request merged**